src/components/pages/home/latest-transactions/latest-transaction.tsx

```diff
@@ -0,0 +1,65 @@
+import AddressLink from "@/components/pages/home/AddressLink";
+import { formatEther } from "@/lib/utils";
+import { Transaction } from "@/lib/types";
+import { ReceiptText } from "lucide-react";
+import { Badge } from "@/components/ui/badge";
+import { formatTimestamp } from "@/lib/utils";
+
+interface LatestTransactionProps {
+  transaction: Transaction;
+}
+
+const LatestTransaction = ({ transaction }: LatestTransactionProps) => {
+  return (
+    <div className="flex items-center gap-4 pt-6">
+      <div className="hidden h-9 w-9 items-center justify-center rounded-lg bg-accent text-accent-foreground 2xl:flex">
+        <ReceiptText className="h-5 w-5" />
+      </div>
+      <div className="flex grow flex-col gap-1">
+        <div className="flex items-center gap-1 text-sm">
+          Tx#{" "}
+          <AddressLink
```

Comment:
The hash is not an address and is unique so id does,'t make sense to use the `AddressLink` component.